CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -74,6 +75,7 @@ local DEFAULT_XBOX_IMAGE = 'rbxasset://textures/ui/Shell/Icons/ROBUXIcon@1080.pn
 local CONTROLLER_CONFIRM_ACTION_NAME = "CoreScriptPurchasePromptControllerConfirm"
 local CONTROLLER_CANCEL_ACTION_NAME = "CoreScriptPurchasePromptControllerCancel"
 local GAMEPAD_BUTTONS = {}
+local BUTTON_TEXT = {}
```

Comment:
this table name doesn't seem descriptive, considering it seems to full of text labels. Maybe just TEXT_OBJECTS?